client/src/main.ts

```diff
@@ -31,5 +32,19 @@ if (import.meta.env.MODE == "production") {
   });
 }
 
+const plausible = createPlausible({
```

Comment:
Von der Idee her, hätte ich die Instanzierung von Plausible in den Folder "config" gepackt.
Die main.ts sollte keine Details enthalten.

Reply:
Ist halt Konvention, dass da zu machen. 
Sentry init ist auch an der gleichen Stelle.

Reply:
Die Konfiguration des Plausible Objektes kann man da aber rausziehen.
Hätte ich für Sentry auch gemacht. 

Das das in der Main eingehangen wird ist vollkommen in Ordnung.

Sonst hast du irgendwann drölfzig Objekte die du in der main.ts konfigurierst.

Reply:
Na gut, habs extrahiert. Sentry bleibt aber drin. Das wird nur komplizierter wenn das woanders hinkommt, wenn man genau hinschaut.